torchvision/transforms/v2/_type_conversion.py

```diff
@@ -80,3 +80,17 @@ def _transform(
 # We changed the name to align them with the new naming scheme. Still, `ToPILImage` is
 # prevalent and well understood. Thus, we just alias it without deprecating the old name.
 ToPILImage = ToImagePIL
+
+
+class ToPureTensor(Transform):
+    """[BETA] Convert all datapoints to pure tensors, removing associated metadata (if any).
+
+    .. v2betastatus:: ToPureTensor transform
+
+    This doesn't scale or change the values, only the type.
+    """
+
+    _transformed_types = (datapoints.Image, datapoints.Video, datapoints.Mask, datapoints.BoundingBoxes)
```

Comment:
Can't we do

```suggestion
    _transformed_types = (datapoints.Datapoint,)
```

? That would make this transform usable for user defined datapoints as well.